scripts/process_data.py

```diff
@@ -28,6 +29,30 @@
 CONSOLE = Console(width=120)
 
 
+def find_tool_feature_matcher_combination(sfm_tool, feature_type, matcher_type):
```

Comment:
It might make sense to move this fn to `process_data_utils.py`

Reply:
done